build.gradle

```diff
@@ -24,7 +24,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.10'
+    ext.kotlin_version = '1.3.30'
```

Comment:
Is there any reason why the version is raised in this PR?

Reply:
No. I've set it to a version were I used Dagger successfully to avoid possible incompatibilities during development. Everything works fine with `1.3.10`. Should I revert it?

Reply:
No, I guess it's fine.